runtime/oti/phelpers.m4

```diff
@@ -135,7 +135,10 @@ define({FUNC_PTR},{r11})
 
 define({CALL_INDIRECT},{
 	ifelse($1,,,{mr FUNC_PTR,$1})
-	bl ._ptrgl[pr]
+	laddr r2,0(FUNC_PTR)
```

Comment:
Can you add a comment that says this is an inlined version of `_ptrgl`?

Reply:
Done.